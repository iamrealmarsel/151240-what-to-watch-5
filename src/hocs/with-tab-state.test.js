import React from "react";
import TestRenderer from "react-test-renderer";
import withTabState from "./with-tab-state";


const MockComponent = () => {
  return <React.Fragment></React.Fragment>;
};

const MockComponentContainer = withTabState(MockComponent);

test(`render withTabState`, () => {
  const tree = TestRenderer
    .create(
        <MockComponentContainer />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
