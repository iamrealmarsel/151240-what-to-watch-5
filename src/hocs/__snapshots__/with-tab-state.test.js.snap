// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withTabState 1`] = `null`;
